### DISCUSSION TITLE
SwiftUI upper toolbar

### DISCUSSION THREAD

**Discussion by kaspesi** [guoyingtao/Mantis#400] [General]
Is there any way to remove the top toolbar that shows up in the demo?

It is the black bar at the top.  I can remove it with `navigationController?.setNavigationBarHidden(true, animated: nil)` but was wondering if there was a better way to do this.
![Screenshot 2024-06-27 at 1 54 38 AM](https://github.com/guoyingtao/Mantis/assets/6432911/872a1cd4-55fd-4486-896e-20d4c83faaa9)
I can change the color but ideally would just like to hide it.  Can remove it by editing the package but not ideal.

Ideally it would look like this example 
![image](https://github.com/guoyingtao/Mantis/assets/6432911/2cb3fc4b-83c1-4e4a-8607-5106fd45ba26)
but didn't see it in the swiftui examples.  Does this mean its not supported?

**Comment by kaspesi** [comment]
Sorry if this is obvious, new to UIKit with SwiftUI

**Reply by guoyingtao** [reply]
In SwiftUI demo, I did not use UINavigationViewController. You mentioned you used `navigationController?.setNavigationBarHidden(true, animated: nil)` to remove the blackbar. Just want to make sure which demo you are using.

**Comment by guoyingtao** [comment]
Hi @kaspesi 
Do you mean Example or SwiftUIExample project for this issue?
Is the first screenshot from an iPhone or iPad?

**Comment by guoyingtao** [comment]
<img width="389" alt="image" src="https://github.com/guoyingtao/Mantis/assets/26723384/faf56e7d-fbb5-4715-b931-c2e28475dc6a">

Is this the effect you want?
